package.json

```diff
@@ -1,35 +1,48 @@
 {
   "name": "fnlint",
-  "version": "0.1.1",
-  "description": "Lint file name formatting",
-  "main": "lib/fnlint.js",
-  "engines": {
-    "node": ">=4.0.0"
-  },
-  "scripts": {
-    "test": "./scripts/test.sh"
-  },
+  "version": "0.0.0-development",
```

Comment:
this is how `semantic-release` works. the version doesn't actually need to be commited, semantic-release just updates it for publishing.